HubSpot.NET.Examples/Program.cs

```diff
@@ -39,14 +43,14 @@ public class Examples
                     });
                 });
         // enable args to be presented from CLI for automated test execution 
-        static void Main(string[] args)
+        static async Task Main(string[] args)
```

Comment:
async here because we make an HTTP request to fetch an image file as part of testing the image upload